entrypoint.sh

```diff
@@ -83,7 +83,7 @@ configure_kernel_module_locking() {
 
     pushd "${mount_path}"
     cp "${grub_cfg}" "${grub_cfg}.orig"
-    sed 's/cros_efi/cros_efi lsm.module_locking=0/g' \
+    sed 's/cros_efi/cros_efi lsm.module_locking=0 loadpin.enabled=0/g' \
```

Comment:
Is this backwards compatible with older kernels? If not, can we put it inside a conditional?

Reply:
I have tested the change against current COS live releases (cos-dev-66-10452-8-0, cos-beta-65-10323-51-0 and cos-stable-64-10176-62-0) and all of them worked. So the change is compatible with older kernel.